fdbclient/admin_client.go

```diff
@@ -201,6 +202,51 @@ func (client *cliAdminClient) runCommand(command cliCommand) (string, error) {
 	return outputString, nil
 }
 
+// CleanupOldLogs removes old fdbcli log files.
+func (client *cliAdminClient) CleanupOldLogs() error {
+	logDir := os.Getenv("FDB_NETWORK_OPTION_TRACE_ENABLE")
+	if logDir == "" {
+		return nil
+	}
+
+	err := filepath.Walk(logDir, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if info.IsDir() {
+			return nil
+		}
+
+		if !strings.HasPrefix(info.Name(), "trace") {
+			return nil
+		}
+
+		// If the file is newer than 1 hour skip it
+		if info.ModTime().Add(1 * time.Hour).Before(time.Now()) {
+			return nil
+		}
+
+		// Files from the lib will have the format:
+		// trace.$IP.1.&randnumber...json
+		// with this regexp we check for the middle part.
+		isCliFile, err := regexp.Compile(`\.1\.\d+`)
```

Comment:
Another more expensive (but more reliable?) approach would be to check the content of the file e.g. if `CLICommandLog` is contained.